website/docs/cdktf/telemetry.mdx

```diff
@@ -14,3 +14,13 @@ Starting with CDK for Terraform 0.6, this information includes a random UUID tha
 CDK for Terraform fingerprints the type of CI used and includes that in telemetry instead of the UUID when CDK for Terraform runs in a Continuous Integration tool, such as GitHub Actions, Jenkins, or GitLab. The only information submitted is the type of CI system, and no unique information about accounts, paths, workspaces, environment variables, or other potentially private information is shared. This data helps the team plan where to focus future efforts, because running infrastructure as code tools in CI is a key workflow we seek to improve.
 
 The information that is sent to Checkpoint is anonymous and cannot be used to identify the user or host. The use of Checkpoint is completely optional and it can be disabled at any time by setting the `CHECKPOINT_DISABLE` environment variable to a non-empty value.
+
+## Crash Reporting
+
+To improve the stability of the [CDK for Terraform CLI](/cdktf/cli-reference/cli-configuration) we also [Sentry](https://sentry.io/) for error reporting.
+
+The error reports send include a stack trace and log events that happened before the error. This information only gets send if there is an error. If no error happens a session is tracked, so that we can calculate the percentage of crash free sessions as a metric.
+
+The error reporting is scoped to the CLI and no environment variables are tracked, limiting the risk of sending secret information by accident.
+
+If you don't want your project to track errors, change the [`sendCrashReports`](/cdktf/create-and-deploy/configuration-file#enable-crash-reporting-for-the-cli) property in your `cdktf.json` file to false.
```

Comment:
This sounds like the default would be on. I think we should mention that we ask first, but if a user wants to disable them later one (after enabling them) they can set it to false.